subprojects/parseq-guava-interop/src/main/java/com/linkedin/parseq/guava/ListenableFutureUtil.java

```diff
@@ -0,0 +1,133 @@
+package com.linkedin.parseq.guava;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.util.concurrent.AbstractFuture;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.linkedin.parseq.BaseTask;
+import com.linkedin.parseq.Context;
+import com.linkedin.parseq.Task;
+import com.linkedin.parseq.promise.Promise;
+import com.linkedin.parseq.promise.SettablePromise;
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.ExecutionException;
+
+
+/**
+ * Utility methods to convert between Parseq {@link Task} and Guava's {@link ListenableFuture}.
+ */
+public class ListenableFutureUtil {
+
+  private ListenableFutureUtil() {
+    // Prevent instantiation.
+  }
+
+  public static <T> Task<T> fromListenableFuture(ListenableFuture<T> future) {
```

Comment:
Is it overkill to create a subproject for this? 
Why not follow the similar convention we used to bridge `CompletionStage`.

Promise.toCompletionStage
Task.fromCompletionStage

Reply:
I also prefer it in separate subproject because Guava might not be so commonly used.

Reply:
I also prefer it in separate subproject because Guava might not be so commonly used.

Reply:
Guava is notorious for backward incompatible changes even in minor
versions. I don't want issues coming up in projects that use parseq unless
they really want to use this guava bridging.